core/src/main/java/org/phoenixframework/liveview/lib/Attribute.kt

```diff
@@ -0,0 +1,50 @@
+package org.phoenixframework.liveview.lib
+
+class Attribute(private var nativeObject: Long) {
+
+    private var _name: String? = null
+
+    /** The name of an attribute */
+    val name
+        get() = _name ?: get_name(nativeObject).also { _name = it }
```

Comment:
You can use lazy initialization for this

```suggestion
    val name: String by lazy {
      get_name(nativeObject)
    }
```